bench/bench_test.go

```diff
@@ -22,20 +22,41 @@ func newRedigo() redigo.Conn {
 	return c
 }
 
+func newRedisPipe(writePause time.Duration) redispipe.Sync {
+	pipe, err := redispipeconn.Connect(context.Background(), "127.0.0.1:6379", redispipeconn.Opts{
+		Logger:     redispipeconn.NoopLogger{},
+		WritePause: writePause,
+	})
+	if err != nil {
+		panic(err)
+	}
+	return redispipe.Sync{S: pipe}
+}
+
+func radixGetSet(client radix.Client, key, val string) error {
+	if err := client.Do(radix.Cmd(nil, "SET", key, val)); err != nil {
+		return err
+	}
+	var out string
+	if err := client.Do(radix.Cmd(&out, "GET", key)); err != nil {
+		return err
+	} else if out != val {
+		return errors.New("got wrong value")
+	}
+	return nil
+}
+
 func BenchmarkSerialGetSet(b *B) {
 	b.Run("radix", func(b *B) {
 		rad, err := radix.Dial("tcp", "127.0.0.1:6379")
 		if err != nil {
 			b.Fatal(err)
 		}
 		defer rad.Close()
+		client := radix.Client(rad)
```

Comment:
Maybe add a small comment why this is here, so this doesn't get "optimized" accidentally? Something like

```suggestion
		// avoid overhead of converting from radix.Conn to radix.Client on each loop iteration
		client := radix.Client(rad)
```